stories/setup/config.js

```diff
@@ -1,11 +1,27 @@
-import { configure } from '@storybook/react';
+import { configure, setAddon } from '@storybook/react';
 import { setOptions } from '@storybook/addon-options';
+import chaptersAddon, { setDefaults } from 'react-storybook-addon-chapters';
+
+// TBD: overrides chapters defaults. Do we want this?
```

Comment:
I think the more default values, the better :)

Reply:
We can at least enable `allowPropTablesToggling`, WDYT?

Reply:
I like the idea of showing props (and source ) but there are few issues with using it here:

1. it only works with Proptypes. I looked into this a bit (when I was looking into to storybook) and there is supposedly a way to get it to work with flow but I've yet to have success with this :/ I'll circle back tho...

2. with the current set up, it looks like this: 
![Alt text](https://monosnap.com/image/kB4nkDttLlphYeAquz5gcGVbOEnPjR.png)

it displays per chapter/section it seems. I think we only need to show it 1 per component, right? So I thought we can use chapters to display not there here but in another way, altho this will be a bit more work... or do you think this is okay?


Regarding the other reasons of overriding the default options:

`allowSourceToggling: false` - same reasons as propsToggling
`showSource: false` - same reasons...
`useTheme: false` - this prevents using !important on styles

Reply:
I added this in a different way (in it's own section so it would not be redundant)..let me know what you think 

note: flow is still an issue which I need to look into. 

Also - thoughts about showing source? (via toggle for each variation of just generic source at the top like I currently have set up for props)